assets/src/blocks/ENForm/FormGenerator.js

```diff
@@ -88,155 +87,153 @@ const TextInput = ({field, onInputChange, onBlur, errors, is_side_style}) => {
     >
       <div
         className="en__field__element en__field__element--text form-group animated-label"
-        style={field.en_type === 'GEN' ? {display: "flex", "flex-direction": "row"} : {display: "block"} }
+        style={field.en_type === 'GEN' ? {display: 'flex', 'flex-direction': 'row'} : {display: 'block'}}
```

Comment:
Here it should be `flexDirection` instead of `'flex-direction'`, currently there is an error: 
```
Warning: Unsupported style property flex-direction. Did you mean flexDirection?
```